mdllog/mdllog_test.go

```diff
@@ -0,0 +1,13 @@
+package mdllog
+
+import (
+	"testing"
+)
+
+func TestAll(t *testing.T) {
+	Trace.Println("Trace logs")
```

Comment:
There are no assertions here.
This is a tough thing to test if we're relying on environment variables, but would be helpful.